site/public/js/ta-grading-rubric.js

```diff
@@ -194,14 +194,15 @@ function ajaxGetComponentRubric(gradeable_id, component_id) {
  * ajax call to get the entire graded gradeable for a user
  * @param {string} gradeable_id
  * @param {string} anon_id
+ * @param {boolean} all_peers
  * @return {Promise} Rejects except when the response returns status 'success'
  */
-function ajaxGetGradedGradeable(gradeable_id, anon_id) {
+function ajaxGetGradedGradeable(gradeable_id, anon_id, all_peers) {
     return new Promise(function (resolve, reject) {
         $.getJSON({
             type: "GET",
             async: AJAX_USE_ASYNC,
-            url: buildCourseUrl(['gradeable', gradeable_id, 'grading', 'graded_gradeable']) + `?anon_id=${anon_id}`,
+            url: buildCourseUrl(['gradeable', gradeable_id, 'grading', 'graded_gradeable']) + `?anon_id=${anon_id}&all_peers=${all_peers}`,
```

Comment:
`all_peers.toString()` to make it the action here explicit as to what should happen with the boolean, to avoid any potential surprises on implicit casting.